src/legacy/core_plugins/visualizations/public/legacy_imports.ts

```diff
@@ -18,12 +18,9 @@
  */
 
 export { PersistedState } from '../../../ui/public/persisted_state';
-export { AggConfig } from '../../../ui/public/agg_types/agg_config';
-export { AggConfigs } from '../../../ui/public/agg_types/agg_configs';
-export {
-  isDateHistogramBucketAggConfig,
-  setBounds,
-} from '../../../ui/public/agg_types/buckets/date_histogram';
+export { IAggConfig } from '../../../ui/public/agg_types';
+export { IAggConfigs } from '../../../ui/public/agg_types';
+export { isDateHistogramBucketAggConfig, setBounds } from '../../../ui/public/agg_types';
```

Comment:
Why we don't have a esLint error here, we do export from  `'../../../ui/public/agg_types' `3 times

![image](https://user-images.githubusercontent.com/20072247/73654215-2a7e8a80-469c-11ea-9383-9f7cd40dd153.png)

Reply:
ah, thanks for pointing this out. they were a byproduct of the way i was grepping to update import paths across kibana -- some of them used to be importing from different paths but have all been changed, and i guess the linter wasn't complaining about it. i went ahead and updated this file.

Reply:
Not sure if this is really worthy to highlight, but there are still some places with the same imports:
`src/legacy/ui/public/visualize/loader/pipeline_helpers/utilities.ts`
`x-pack/legacy/plugins/rollup/public/legacy.ts`
`x-pack/legacy/plugins/rollup/public/legacy_imports.ts`